e2e/test_helpers.go

```diff
@@ -636,3 +645,48 @@ func validateSSHConnectivity(ctx context.Context, s *Scenario) error {
 	s.T.Logf("SSH connectivity to %s verified successfully", s.Runtime.VMPrivateIP)
 	return nil
 }
+
+func runScenarioGPUNPD(t *testing.T, vmSize, location, vmSKU string) *Scenario {
```

Comment:
`vmSize` and `vmSKU` are missleading. I think one of them should be something like `k8sSystemSKU`?

Also, I would probably just add new variable `ClusterKubenetSouthcentralus`.
It seems like the SKU is location specific anyway? Not sure if you gain anything by making it configurable. But it adds complexity

Reply:
> Also, I would probably just add new variable ClusterKubenetStandard_D2s_v3 instead.

I am not sure what do you mean by add a variable? Add a variable where? 

> It seems like the SKU is location specific anyway?

Yep the SKU is location specific until we have capacity issues for the default one that we use for the whole e2e test run.

> Not sure if you gain anything by making it configurable. But it adds complexity

I am not sure I understand this either. Could you elaborate?